OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -236,5 +274,27 @@ void Explode(World world)
 
 			args.Weapon.Impact(Target.FromPos(pos), args.SourceActor, args.DamageModifiers);
 		}
+
+		bool AnyValidTargetsInRadius(World world, WPos pos, WDist radius, Actor firedBy, bool checkTargetType)
```

Comment:
I have an idea about that, but I probably won't get around to testing it until the weekend.